docs/fabric-native-components-ios.md

```diff
@@ -0,0 +1,184 @@
+---
+id: fabric-native-components-ios
+title: 'Fabric Native Components: iOS'
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import constants from '@site/core/TabsConstants';
+
+## Using RTNCenteredText to your Application
+
+You can [manually run](the-new-architecture/codegen-cli) the Codegen, however it's simpler to use the application you're going to demo the component in to do this for you.
+
+### 1. Run Codegen through CocoaPods
+
+```bash
+cd ios
+bundle install
+bundle exec pods install
+```
+
+Importantly you will see logging output from Codegen, such as:
+
+```log
+...
+[Codegen] >>>>> Searching for codegen-enabled libraries in react-native.config.js
+...
+[Codegen] Processing RTNCenteredTextSpecs
+[Codegen] Searching for podspec in the project dependencies.
+[Codegen] Supported Apple platforms: ios for RTNCenteredTextSpecs
+[Codegen] Generating Native Code for FBReactNativeSpec - ios
+// highlight-next-line
+[Codegen] Generated artifacts: /Users/me/Demo/ios/build/generated/ios
+```
+
+:::warning
+You should be careful about committing this generated code to your repository. The generated code is specific to each version of React Native. Use npm [peerDependencies](https://nodejs.org/en/blog/npm/peer-dependencies) to restrict compatibility with version of React Native.
+:::
```

Comment:
this can probably be removed: the gitignore already ignores the `ios/build` folder where the code is generated by Codegen.